handlers/api.py

```diff
@@ -446,6 +446,8 @@ def get(self, type, resource=''):
             shake = Shake.get('name=%s and deleted=0', resource)
         elif type == 'shake_id':
             shake = Shake.get('id=%s and deleted=0', resource)
+        elif type == 'shake_user':
+            shake = User.get('name=%s', resource).shake()
```

Comment:
Maybe

    user = User.get('name=%s and deleted=0', resource)
    if user is not None:
        shake = user.shake()